src/app/components/Detail/WorkDetail.jsx

```diff
@@ -45,9 +44,10 @@ class WorkDetail extends React.Component {
 
   componentDidMount() {
     const { query, hash } = this.props.location;
-    const workId = query && query.workId;
     this.setState({ shouldShowAll: query && query.showAll ? (query.showAll === 'true') : true });
-    this.loadWork(workId, hash, this.boundActions);
+    if (query) {
```

Comment:
If `query` is missing should an error be thrown, or a redirect happen back to the landing page?